Library/UseCases/SearchFiltersUseCase.swift

```diff
@@ -224,15 +228,33 @@ private func filterPills(
       SearchFilterPill(
         isHighlighted: state != SearchFiltersUseCase.defaultStateOption,
         filterType: .projectState,
-        // FIXME: MBL-2218 Turn the state into a user-readable title.
-        buttonType: .dropdown(state.rawValue)
+        // FIXME: MBL-2232 Final translations
+        buttonType: .dropdown(hasState ? state.title : "Project status")
       )
     )
   }
 
   return pills
 }
 
+extension DiscoveryParams.State {
+  // FIXME: MBL-2232 Final translations
+  var title: String {
+    switch self {
+    case .all:
+      "All"
+    case .live:
+      "Live"
+    case .successful:
+      "Successful"
+    case .late_pledge:
+      "Late Pledge"
+    case .upcoming:
+      "Upcoming"
+    }
```

Comment:
these are also getting pulled in with my [Tracking Number Translations PR](https://github.com/kickstarter/ios-oss/pull/2389)